### DISCUSSION TITLE
Binary package for debian?

### DISCUSSION THREAD

**Discussion by deltazefiro** [MoganLab/mogan#439] [Q&A]
I looked through the installation guide, but I can't find any deb package to download or PPA source.  
Have I missed something? Or I have to compile it myself?  
Thanks.

**Comment by YiqiXu** [comment]
Would you mind reposting this question on the [forum](http://forum.texmacs.cn)? 
The Discussions section on GitHub will soon be closed.

**Comment by da-liii** [comment]
Let me close it now!

**Comment by JackYansongLi** [comment]
You can download the debian image in here: https://liiistem.cn/install.html
